.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+  pull_request:
+      types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
I am wondering if we can add `3.9` to the python-version. In the [conda environment file, we defined python version as](https://github.com/eWaterCycle/ewatercycle/blob/b90f46bc23ce5aed5003e317d22f1aff3fb2f9a7/environment.yml#L7)>=3.6.  I am not sure if these two files should be consistent regarding the python version.

Reply:
I took the Python version listed in https://github.com/eWaterCycle/ewatercycle/blob/dcdd5e393bf993c449a524817120cbafacfec1f7/setup.py#L39-L41
that is my source of truth for supported Python versions. 

Indeed, we should keep the versions in sync